src/combat.ts

```diff
@@ -320,11 +322,14 @@ export class Macro extends StrictMacro {
     // Determine stasis item to use
     // Garbo already gets a seal tooth at the start of the day, so that's around always
     let stasisItem = $item`seal tooth`;
-    if (retrieveItem($item`facsimile dictionary`)) {
-      // The dictionaries are nicer though as they don't hurt the monster
-      stasisItem = $item`facsimile dictionary`;
-    } else if (retrieveItem($item`dictionary`)) {
-      stasisItem = $item`dictionary`;
+    if (!globalOptions.checkedForDictionaries) {
```

Comment:
stasisItem will only ever be set to a dictionary the first time this block is entered now